test/extended/router/external_certificate.go

```diff
@@ -46,7 +47,7 @@ var _ = g.Describe("[sig-network][OCPFeatureGate:RouteExternalCertificate][Featu
 	defer g.GinkgoRecover()
 	var (
 		oc            = exutil.NewCLIWithPodSecurityLevel("router-external-certificate", admissionapi.LevelBaseline)
-		helloPodPath  = exutil.FixturePath("..", "..", "examples", "hello-openshift", "hello-pod.json")
+		helloPodPath  = exutil.FixturePath("testdata", "cmd", "test", "cmd", "testdata", "hello-openshift", "hello-pod.json")
```

Comment:
does this change switch out the image?

Reply:
Yes, it will change from https://github.com/openshift/origin/blob/main/examples/hello-openshift/hello-pod.json to https://github.com/openshift/origin/blob/main/test/extended/testdata/cmd/test/cmd/testdata/hello-openshift/hello-pod.json file. 


So, changing the image from `openshift/hello-openshift` to `registry.k8s.io/e2e-test-images/agnhost:2.53`

Reply:
oh lol ok that's the path got it..